addons/website/models/website.py

```diff
@@ -1340,3 +1341,235 @@ def _get_cached_values(self):
+                    similar = ["%s <%%%% %s" % (unaccent("%(search)s"), unaccent(field)) for field in fields]
+                else:
+                    similarities = ["word_similarity(%s, %s)" % ("%(search)s", field) for field in fields]
+                    similar = ["%s <%%%% %s" % ("%(search)s", field) for field in fields]
+                best_similarity_str = 'GREATEST(%s)' % ', '.join(similarities)
+
+                query = """
+                    SELECT id, {best_similarity_str} AS _best_similarity
+                    FROM {table}
+                    WHERE {where}
+                    ORDER BY _best_similarity desc
+                    LIMIT 1000
+                """.format(
+                    best_similarity_str=best_similarity_str,
+                    table=model._table,
+                    where=' OR '.join(similar),
+                )
+                self.env.cr.execute(query, {'search': search})
+                ids = set([row[0] for row in self.env.cr.fetchall()])
+                if self.env.lang:
+                    if self.env.registry.has_unaccent:
+                        unaccent = get_unaccent_wrapper(self.env.cr)
+                        similarity_str = "word_similarity(%s, %s)" % (unaccent(search_term), unaccent('value'))
+                        similar = "%s <%%%% %s" % (unaccent(search_term), unaccent('value'))
+                    else:
+                        similarity_str = "word_similarity(%s, %s)" % (search_term, 'value')
+                        similar = "%s <%%%% %s" % (search_term, 'value')
+                    names = ['%s,%s' % (escape_psql(model._name), escape_psql(field)) for field in fields]
+                    query = """
+                        SELECT res_id, {similarity_str} AS _similarity
+                        FROM ir_translation
+                        WHERE {where}
+                        AND lang = %s
+                        AND name = ANY (%s)
+                        AND type = 'model'
+                        ORDER BY _similarity desc
+                        LIMIT 1000
+                    """.format(
+                        similarity_str=similarity_str,
+                        where=similar,
+                    )
+                    self.env.cr.execute(query, (self.env.lang, names))
+                    for row in self.env.cr.fetchall():
+                        ids.add(row[0])
+                domain = search_detail['base_domain'].copy()
+                domain.append([('id', 'in', list(ids))])
+                domain = AND(domain)
+                records = model.search_read(domain, fields, limit=limit)
+                for record in records:
+                    for field, value in record.items():
+                        if isinstance(value, str):
+                            value = value.lower()
+                            if field == 'arch_db':
+                                value = self._search_text_from_html(value)
```

Comment:
`website` does not extend the website searchable mixin tho does it?

Reply:
Turned in to a tool method, and revived the arch_db search